Airports/EGBB/Sids.txt

```diff
@@ -5,8 +5,8 @@ SID:EGBB:15:CPT2Y:BBS02 BBS04 BBS06 BBS13 BBS17 CPT
 SID:EGBB:15:DTY2Y:BBS02 BBS04 BBS06 BBE09 BBE12 BBE16 BBE19 DTY
 SID:EGBB:15:LUVUM1Y:BBE23 BBE24 BBE25 BBN11 BBN12 LUVUM
 SID:EGBB:33:LUVUM1M:BBN30 BBN32 LUVUM
-SID:EGBB:15:LUXTO15:D147B D165D LUXTO
-SID:EGBB:15:MOSUN15:D147B D165D MOSUN
+SID:EGBB:15:LUXTO15:D144B D153D LUXTO
+SID:EGBB:15:MOSUN15:D144B D153D MOSUN
```

Comment:
For me, this is still showing as track 147, then 165?
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/13762210/96f52c7b-4c5c-4075-a6e7-bc4cc94d6c94)